docs/ember-intl/package.json

```diff
@@ -29,12 +29,12 @@
     "test": "ember test --test-port=0"
   },
   "devDependencies": {
-    "@babel/core": "^7.26.10",
+    "@babel/core": "^7.27.1",
     "@ember/optional-features": "^2.2.0",
     "@ember/string": "^3.1.1",
     "@ember/test-helpers": "^5.2.1",
-    "@embroider/compat": "^3.8.5",
-    "@embroider/core": "^3.5.5",
+    "@embroider/compat": "^3.9.0",
+    "@embroider/core": "^3.5.6",
```

Comment:
At the moment, `@embroider/compat@4.x`, `@embroider/core@4.x`, and `@embroider/router@3.x` don't support apps with Webpack.